provisioning/AWS/aws-php-storage-driver-common-resources.yaml

```diff
@@ -73,6 +58,43 @@ Resources:
       ThumbprintList:
         - 6938fd4d98bab03faadb97b34396831e3780aea1
 
+  S3Bucket:
+    Type: "AWS::S3::Bucket"
+    Properties:
+      BucketName: ci-storage-driver-common
+      AccessControl: Private
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+      LifecycleConfiguration:
+        Rules:
+          - Id: DeleteExpiredObjects
+            Status: Enabled
+            ExpirationInDays: 30
+            NoncurrentVersionExpirationInDays: 90
+
+  S3BucketPolicy:
+    Type: "AWS::S3::BucketPolicy"
```

Comment:
To opravveneni se bude resit jinak ne pomoci bucket policy. U te IAM role ktera bude pristupovat do bucketu priradis ty prava

Reply:
Kdyztak ukazu v pondeli

Reply:
bucket policy se tedy vubec nebude nastavovat, pouziva se vyjimecne spis pro cross account pristupy apod

Reply:
jj nieco som testoval  malo by to byt asi v tych policies ako je tam to DownloadDrivers ze ?
```
Policies:
        - PolicyName: DownloadDrivers
          PolicyDocument:
            Version: "2012-10-17"
            Statement:
              - Action:
                  - s3:ListBucket
                  - s3:GetObject
                Resource:
                  - arn:aws:s3:::keboola-drivers/*
                  - arn:aws:s3:::keboola-drivers
                Effect: Allow
        - PolicyName: FileStorage
          PolicyDocument:
            Version: "2012-10-17"
            Statement:
              - Action:
                  - s3:ListBucket
                  - s3:GetObject
                  - s3:PutObject
                Resource:
                  - arn:aws:s3:::ci-storage-driver-common/*
                  - arn:aws:s3:::ci-storage-driver-common
                Effect: Allow
```